test/ranking.jl

```diff
@@ -1,31 +1,45 @@
 using StatsBase
+using Missings
 using Compat
 using Compat.Test
 
 a = [1.0, 2.0, 2.0, 3.0, 4.0, 4.0, 4.0, 5.0]
 x = [3.0, 1.0, 2.0, 4.0, 4.0, 2.0, 5.0, 4.0]  # x is a permutated version of a
+xm = [3.0, 1.0, 2.0, 4.0, 4.0, 2.0, 5.0, 4.0, missing]
 s = ["c", "a", "b", "d", "d", "b", "e", "d"] # s is a vector of strings ordered like x
 
 @test ordinalrank(a) == [1, 2, 3, 4, 5, 6, 7, 8]
 @test ordinalrank(x) == [4, 1, 2, 5, 6, 3, 8, 7]
+@test collect(skipmissing(ordinalrank(xm))) == [4, 1, 2, 5, 6, 3, 8, 7]
+@test ismissing(ordinalrank(xm)[end])
```

Comment:
It would be good to check that the returned value has the expected values apart from the last one. Use `isequal` to compare it with the expected array. Same below.